prime-dotnet-webapi/Services/interfaces/ISiteService.cs

```diff
@@ -51,5 +51,6 @@ public interface ISiteService
         Task<SiteRegistrationNoteViewModel> GetSiteRegistrationNoteAsync(int siteId, int siteRegistrationNoteId);
         Task<IEnumerable<int>> GetNotifiedSiteIdsForAdminAsync(ClaimsPrincipal user);
         Task<bool> SiteExists(int siteId);
+        Task<bool> PecExistsInNonHealthAuthoritySites(string pec);
```

Comment:
I would advise against putting business logic in the name of a method. I makes the name really long and it means that if we want to change the business logic later we have to alter the name. I would prefer something like `PecIsAvailibleAsync` or `ValidatePecAsync` or `CheckPecConflictAsync` or something like that.